src/common/api/operations.ts

```diff
@@ -46,6 +48,21 @@ export interface CommentOptions {
 export type RewardType = "default" | "sp" | "dp";
 
 export const formatError = (err: any): string => {
+
+    const strErr = err.toString();
+
+    if (/You may only post once every/.test(strErr)) {
+        return _t("chain-error.min-root-comment");
+    } else if (/Your current vote on this comment is identical/.test(strErr)) {
+        return _t("chain-error.identical-vote");
+    } else if (/Please wait to transact, or power up/.test(strErr)) {
+        return _t("chain-error.insufficient-resource");
+    } else if (/Cannot delete a comment with net positive/.test(strErr)) {
+        return _t("chain-error.delete-comment-with-vote");
+    } else if(/comment_cashout/.test(strErr)){
+        return _t("chain-error.comment-cashout");
+    }
```

Comment:
There is also error on voting old posts, would be good to add that as well. @talhasch